sdk/dotnet/Pulumi.Docker.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Authors>Pulumi Corp.</Authors>
+    <Company>Pulumi Corp.</Company>
+    <Description>A Pulumi package for interacting with Docker in Pulumi programs.</Description>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageProjectUrl>https://pulumi.io</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/pulumi/pulumi-docker</RepositoryUrl>
+    <PackageIcon>logo.png</PackageIcon>
+
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1701;1702;1591</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="version.txt" />
+    <Content Include="version.txt" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi" Version="1.7.0-preview-alpha.1574743805" />
```

Comment:
I'll need to change this to `1.6.1-preview` or `1.7.0-preview` whenever we have them